napari/_vendor/cpython/functools.py

```diff
@@ -0,0 +1,57 @@
+from _thread import RLock
```

Comment:
since you're not bringing in `_thread`, this probably needs to be `from threading import RLock`

Reply:
changed